src/shared/components/fixed-position-container.jsx

```diff
@@ -53,6 +53,12 @@ export function mapStateToProps(state, ownProps) {
       width: "100%",
       height: "100%"
     };
+
+    if (state.viewMode === "REPORT_VIEW") {
+      style.position = "static";
+      style.border = "none";
+      style.overflow = "visible";
+    }
```

Comment:
I suspect it is possible to do this with pure css / emotion, which is probably preferable.